hack/start-registry-proxies.sh

```diff
@@ -14,11 +14,6 @@ docker run -d -p 5001:5000 \
     --restart always \
     --name registry-registry.k8s.io registry:2
 
-docker run -d -p 5002:5000 \
```

Comment:
we no longer have images from `quay.io` (it was Flannel, but we mirror it under `ghcr.io`)